concept/type_/type_manager/validation/operation_time_validation.rs

```diff
@@ -1363,621 +846,643 @@ impl OperationTimeValidation {
+        value_type: Option<ValueType>,
     ) -> Result<(), SchemaValidationError> {
-        let mut validation_errors = vec![];
-
-        let cardinality_declared =
-            capability.get_cardinality_declared(snapshot, type_manager).map_err(SchemaValidationError::ConceptRead)?;
-        let cardinality = if let Some(cardinality_declared) = cardinality_declared {
-            // Does not check that declared cardinality narrows updated inherited one! Should be checked in a separate validation
-            cardinality_declared
-        } else if let Some(overridden_capability) = &overridden_capability {
-            overridden_capability.get_cardinality(snapshot, type_manager).map_err(SchemaValidationError::ConceptRead)?
+        if values.valid(value_type.clone()) {
+            Ok(())
         } else {
-            capability.get_default_cardinality(snapshot, type_manager).map_err(SchemaValidationError::ConceptRead)?
-        };
-
-        let updated_cardinalities =
-            Self::get_transitively_updated_cardinalities(snapshot, type_manager, capability.clone(), cardinality)
-                .map_err(SchemaValidationError::ConceptRead)?;
-        let updated_overrides = HashMap::from([(capability.clone(), overridden_capability.clone())]);
-        validate_capabilities_cardinality::<CAP>(
-            type_manager,
-            snapshot,
-            capability.object(),
-            &updated_cardinalities,
-            &updated_overrides,
-            &mut validation_errors,
-        )
-        .map_err(SchemaValidationError::ConceptRead)?;
-
-        if let Some(error) = validation_errors.first() {
-            return Err(error.clone());
+            Err(SchemaValidationError::InvalidValuesArgumentsForValueType(values, value_type))
         }
+    }
 
-        // Preserve inheritance ordering in validations
-        // (expect errors from inheritance levels closer to the changed capability)
-        let mut checked_object_types: HashSet<CAP::ObjectType> = HashSet::new();
-        let overriding_capabilities = TypeReader::get_overriding_capabilities_transitive(snapshot, capability.clone())
-            .map_err(SchemaValidationError::ConceptRead)?;
-        for overriding_capability in overriding_capabilities {
-            let not_checked = checked_object_types.insert(overriding_capability.object());
-            if !not_checked {
-                continue;
-            }
-
-            validate_capabilities_cardinality::<CAP>(
-                type_manager,
-                snapshot,
-                overriding_capability.object(),
-                &updated_cardinalities,
-                &updated_overrides,
-                &mut validation_errors,
-            )
-            .map_err(SchemaValidationError::ConceptRead)?;
-
-            if let Some(error) = validation_errors.first() {
-                return Err(error.clone());
-            }
-        }
+    // TODO: Capabilities constraints narrowing checks are currently disabled
```

Comment:
I'm commenting out these BDD tests as well (instead of deleting them completely)
I have a task for it, and I'll prepare a plan for these validations, estimate it and we can prioritize it after the first alpha.